lua/weapons/arccw_base/shared.lua

```diff
@@ -183,13 +183,15 @@ SWEP.EnterBipodSound = "weapons/arccw/m249/m249_coverdown.wav"
 SWEP.ExitBipodSound = "weapons/arccw/m249/m249_coverup.wav"
 SWEP.SelectUBGLSound =  "weapons/arccw/ubgl_select.wav"
 SWEP.ExitUBGLSound = "weapons/arccw/ubgl_exit.wav"
+SWEP.ShootSoundWorldCount = 0 -- every shot, play the fire sound this many more times in the world (adds volume), unless suppressed
```

Comment:
strange feature

Reply:
perhaps, but it works, is disabled by default, and i need it for the jmod guns

Reply:
can't you do an hook_addshootsound and copy the values from the hook?